apps/lib/opt.c

```diff
@@ -359,6 +359,9 @@ void print_format_error(int format, unsigned long flags)
 int opt_cipher(const char *name, const EVP_CIPHER **cipherp)
 {
     *cipherp = EVP_get_cipherbyname(name);
+    if (*cipherp != NULL)
```

Comment:
Not sure I understand the order here.. Didnt we agree on the reverse order?
See https://github.com/openssl/openssl/issues/14198

Reply:
We did, but it breaks tests, at least because some output changed.  Waiting for the "official names" PR to get merged and then we'll look at this.

Reply:
I've reopened #14198 so we do not forget.